content/docs/rendering-elements.md

```diff
@@ -8,68 +8,69 @@ prev: introducing-jsx.html
 next: components-and-props.html
 ---
 
-Elements are the smallest building blocks of React apps.
+एलिमेंट्स रिएक्ट ऐप्स के सबसे छोटे बिल्डिंग ब्लॉक हैं।
 
-An element describes what you want to see on the screen:
+एक एलिमेंट वर्णन करता है कि आप स्क्रीन पर क्या देखना चाहते हैं:      
 
 ```js
 const element = <h1>Hello, world</h1>;
 ```
 
-Unlike browser DOM elements, React elements are plain objects, and are cheap to create. React DOM takes care of updating the DOM to match the React elements.
+ब्राउज़र DOM एलेमेंट्स के विपरीत, React एलेमेंट्स प्लेन ऑब्जेक्ट है और इन्हें बनाना आसान है। React DOM React एलेमेंट्स के अनुसार DOM को अपडेट रखता है।
```

Comment:
`एलेमेंट्स` => `एलिमेंट्स`

Any thoughts?

Reply:
done

Reply:
There is one more you have missed.
I think you should do CTLR+F and Replace All

Reply:
done